app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,150 @@
+                  if (selectedContactIds.length === contactIds.length) {
+                    setSelectedContactIds([]);
+                  } else {
+                    setSelectedContactIds(contactIds);
+                  }
+                }}
+              />
+            </th>
+            <th className="text-left">Name</th>
+            <th className="text-left">Email</th>
+            <th className="text-center">Department</th>
+            <th className="text-center">Contact Number</th>
+            <th className="text-center">Add to Basecamp</th>
+          </tr>
+        </thead>
+        <tbody>
+          {contacts.map(contact => (
+            <tr
+              key={contact.id}
+              className={"cursor-pointer bg-white hover:bg-gray-50"}
+            >
+              <td>
+                <Checkbox
+                  checked={selectedContactIds.includes(contact.id)}
+                  onClick={event => {
+                    event.stopPropagation();
+                    const index = selectedContactIds.indexOf(contact.id);
+
+                    if (index > -1) {
+                      setSelectedContactIds([
+                        ...selectedContactIds.slice(0, index),
+                        ...selectedContactIds.slice(index + 1),
+                      ]);
+                    } else {
+                      setSelectedContactIds([
+                        ...selectedContactIds,
+                        contact.id,
+                      ]);
+                    }
+                  }}
+                />
+              </td>
+              <td>
+                <div className="flex flex-row items-center justify-start text-gray-900">
+                  <Avatar
+                    size={36}
+                    bgClassName="bg-blue-300 mr-3"
+                    contact={{ name: contact.initial }}
+                  />
+                  {contact.initial}
+                </div>
+              </td>
+              <td>{contact.email}</td>
+              <td className="text-center">{contact.department}</td>
+              <td className="text-center">{contact.contact_number}</td>
+              <td className="flex justify-center">
+                <div>
+                  {contact.add_to_basecamp && (
+                    <Checkbox name="Basecamp" checked />
+                  )}
+
+                  {!contact.add_to_basecamp && <Checkbox name="Basecamp" />}
+                </div>
+              </td>
+              <td>
+                <Tooltip content="Edit" position="bottom">
+                  <Button style="icon" icon="ri-pencil-line" />
+                </Tooltip>
+              </td>
+              <td>
+                <Tooltip content="Delete" position="bottom">
+                  <Button
+                    style="icon"
+                    icon="ri-delete-bin-line"
+                    onClick={() => setDeletingRecord(contact.id)}
+                  />
+                </Tooltip>
```

Comment:
Wrap them inside a single `<td></td>`. This will correspond to the empty `th` created earlier.